Music/music.md

```diff
@@ -25,6 +25,10 @@ description: List of sources for getting all genres of Music.
 
 [**Unreleased Disco**](https://unreleased.me/) - Huge archive of Unreleased Hip-Hop/Pop Songs from various Artists.
 
+[**R/RipRequest**](https://www.reddit.com/r/riprequests/) - A sub-reddit for requesting songs (most flacs) **_Is currently private but you can request access_**
```

Comment:
change sub-reddit to subreddit, change "requesting songs (most flacs)"  to "requesting lossless rips of albums"